### PR TITLE
Fix issue where special chars cause PatternSyntaxException

### LINKED ISSUE #2433: Regex error when running unexpected mv command
### Details
I was **able** to reproduce my issue on a freshly setup and up-to-date server with the latest version of Multiverse plugins with no other plugins and with no kinds of other server or client mods.

**Description**  
https://pastebin.com/raw/hFCMPrB7

**Steps to reproduce**  
run an invalid command like `mv hel[p`

### PR DESCRIPTION
**Pull request by benwoo1110**
Escape regex special chars for help command, fixes #2433

### CODE REVIEW

**Comment by nicegamer7** [comment]
Does this break filtering?

**Comment by benwoo1110** [comment]
From my testing no, anyways it's going to be completely changed in ACF as well.

**Comment by benwoo1110** [comment]
@nicegamer7 can we merge this or smt? (want to delete my old mv forks 😅)

File: src/main/java/com/onarandombox/MultiverseCore/commands/PaginatedCommand.java [hunk 1]
```diff
@@ -40,12 +42,23 @@ protected void setItemsPerPage(int items) {
...
     protected abstract List<T> getFilteredItems(List<T> availableItems, String filter);
 
+    /**
+     * Escape regex special characters from filter
+     *
+     * @param filter The filter-{@link String}.
+     * @return String with regex characters escaped
+     */
+    protected String cleanFilter(String filter) {
+        return REGEX_SPECIAL_CHARS.matcher(filter.toLowerCase()).replaceAll("\\\\$0");
```
Comment (nicegamer7) [inline src/main/java/com/onarandombox/MultiverseCore/commands/PaginatedCommand.java:59]: Any reason you went to lower case here?

Comment (benwoo1110) [reply src/main/java/com/onarandombox/MultiverseCore/commands/PaginatedCommand.java]: brain goes brrrr

**Review by nicegamer7 — APPROVED** [review]
I was originally hesitant to merge this because I wasn't sure if the filter was meant to allow users to search using regex, but I came to the realization that I don't think anyone is searching for a command using regex lol.

**Comment by benwoo1110** [comment]
nah, it was just sole to fix a regex parsing issue

**Pull request merged**